### ISSUE TITLE
Can't invoke `Show Terminal Accessibility Help` with keybindings

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#172196]
Testing #171862

No matter whether I use `Shift+F1` or set it to some other keybinding, I cannot seem to invoke this command with a keyboard shortcut--only from the command palette.

**Comment by meganrogge** [comment]
I cannot repro on mac, windows, or linux, could you pls run `Developer: Toggle Keyboard Shortcuts Troubleshooting` and provide me with the logs?

The terminal has to be focused btw for it to work

**Comment by joyceerhl** [comment]
Yeah the terminal was focused in my testing. Weirdly enough, seems to work now--definitely wasn't working before though.

**Comment by gregvanl** [comment]
I've run into this as well. Here's the output when turned on Toggle Keyboard Shortcuts Troubleshooting:

2023-01-24 19:45:48.902 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:48.925 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:48.925 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:48.925 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:48.925 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.404 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.404 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.404 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.405 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.405 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.435 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.435 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.435 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.435 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.435 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.482 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.482 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.482 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.483 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.483 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.513 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.514 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.514 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.514 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.514 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.544 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.545 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.545 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.545 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.545 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.577 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.577 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.577 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.577 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.578 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.608 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.608 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.608 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.608 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.609 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.641 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.642 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.642 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.643 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.643 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.673 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.673 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.674 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.674 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.675 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.703 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.703 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.703 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.703 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.704 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.735 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.736 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.736 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.736 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.736 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.767 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.767 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.767 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.768 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.768 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.798 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.798 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.798 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.799 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.799 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.844 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.844 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.845 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.845 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.845 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.877 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.877 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.878 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.878 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.879 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.907 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.908 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:49.909 [info] [KeybindingService]: / Received  keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 16, key: Shift
2023-01-24 19:45:49.909 [info] [KeybindingService]: | Converted keydown event - modifiers: [shift], code: ShiftLeft, keyCode: 4 ('Shift')
2023-01-24 19:45:49.910 [info] [KeybindingService]: \ Keyboard event cannot be dispatched in keydown phase.
2023-01-24 19:45:49.916 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:49.917 [info] [KeybindingService]: | Resolving shift+F1
2023-01-24 19:45:49.917 [info] [KeybindingService]: \ From 2 keybinding entries, matched workbench.action.terminal.showAccessibilityHelp, when: terminalFocus && terminalProcessSupported, source: built-in.
2023-01-24 19:45:50.063 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:50.063 [info] [KeybindingService]: | Resolving shift+F1
2023-01-24 19:45:50.063 [info] [KeybindingService]: \ From 2 keybinding entries, matched workbench.action.terminal.showAccessibilityHelp, when: terminalFocus && terminalProcessSupported, source: built-in.
2023-01-24 19:45:50.483 [info] [KeybindingService]: / Soft dispatching keyboard event
2023-01-24 19:45:50.483 [info] [KeybindingService]: \ Keyboard event cannot be dispatched
2023-01-24 19:45:50.484 [info] [KeybindingService]: + Ignoring single modifier shift due to it being pressed together with other keys.

**Comment by gregvanl** [comment]
This was on my Surface with Windows 11. I wonder if it has to do with needing to press the Windows function key to get my F1 key? I need to do Shift + Function + F1 as without the Function key, F1 is the screen dimmer

**Comment by gregvanl** [comment]
Alt+Function+F1 works for showing the regular accessibility help

**Comment by meganrogge** [comment]
> I need to do Shift + Function + F1 as without the Function key, F1 is the screen dimmer.

This is expected as to use F1, the function key needs to be activated.

> Alt+Function+F1 works for showing the regular accessibility help. I wonder what that code is doing to work?

Alt in the terminal does funny things on Windows/Linux, which is why we went with `Shift` instead here

**Comment by meganrogge** [comment]
On Windows, when I set the keybinding to `alt+f1`, it's not a valid chord and the result from the keybinding service is `workbench.action.showCommands`

![image (5)](https://user-images.githubusercontent.com/29464607/214587199-55c3a699-85ba-4f07-84a0-03c149b4cc81.png)

**Comment by Tyriar** [comment]
shift+f1 isn't working for me
